Casks/d/domzilla-caffeine.rb

```diff
@@ -20,16 +20,16 @@
     sha256 "5d1127fe3a5be772c61b9685915bd244b2e47b4607242b0d3b9cc517f5269c43"
 
     livecheck do
-      url "https://www.caffeine-app.net/download"
-      strategy :header_match
+      url "https://dr-caffeine-mac.s3.amazonaws.com/appcast.xml"
+      strategy :sparkle, &:short_version
     end
   end
 
   url "https://dr-caffeine-mac.s3.amazonaws.com/Caffeine_#{version}.zip",
       verified: "dr-caffeine-mac.s3.amazonaws.com/"
   name "Caffeine"
   desc "Utility that prevents the system from going to sleep"
-  homepage "https://www.caffeine-app.net/"
+  homepage "https://github.com/domzilla/Caffeine"
```

Comment:
```suggestion
  homepage "https://www.caffeine-app.net/"
```

I may be overlooking something but www.caffeine-app.net links to the GitHub repository and https://www.caffeine-app.net/download/ redirects to https://dr-caffeine-mac.s3.amazonaws.com/Caffeine_1.4.3.zip, so it seems like the correct homepage. The GitHub repository doesn't have tags for any versions after 1.1.3 (and no releases), so it may be less useful than www.caffeine-app.net (which lists the latest version for each supported macOS version).

Reply:
Thank you, you are correct! I'm running Pi-hole on my local network, and it seems that the domain `caffeine-app.net` is being blocked by the filter list [`RPiList-Malware`](https://v.firebog.net/hosts/RPiList-Malware.txt) for some reason.